src/world.rs

```diff
@@ -0,0 +1,246 @@
+//! Worlds (a group of dimensions).
+//!
+//! This module is a WORK IN PROGRESS.
+
+use std::io::{self, Write};
+use std::net::TcpStream;
+use std::ops::Sub; // Sub for Timespec
+use std::thread::sleep;
+use std::time::Duration;
+
+use packet::{ChunkMeta, PacketRead, PacketWrite, Protocol};
+use types::consts::*;
+use types::{Chunk, ChunkColumn, Var};
+use util::ReadExactExt;
+
+use rand;
+use time;
+
+const PACKET_NAMES: [&'static str; 26] = [
```

Comment:
We probably need to do it in a follow-up PR because there were some weird types not having Debug so we have to manually implement it.